hydra_agent/agent.py

```diff
@@ -102,7 +105,28 @@ def get(self, url: str = None, resource_type: str = None,
             if response.json()['@type'] in self.api_doc.parsed_classes:
                 return response.json()
             else:
-                return response.json()['members']
+                if follow_partial_links:
+                    """
+                    Iterator will be returned.
```

Comment:
You should move this to the function docstrings. It's not a good approach to have big chunks of documentation between functions.